test/e2e/specs/editor/various/fullscreen-mode.spec.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Full Screen', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllPosts();
+	} );
+
+	test( 'should open the fullscreen mode from the more menu', async ( {
+		page,
+	} ) => {
+		// Open Options
+		await page.locator( ' [aria-label="Options"]' ).click();
+
+		// Verify All options are visible
+		page.locator( '.interface-more-menu-dropdown__content' );
+
+		// Choose Full Screen Mode
+		await page
+			.locator( 'text=/Fullscreen modeWork without distraction/' )
+			.click();
```

Comment:
```suggestion
		await page
			.locator( 'role=menuitemcheckbox', { hasText: 'Fullscreen mode' } )
			.click();
```

Nitpick: Let's use a role-based selector.